File: MyEvent_Xamarin/MyEvent.WebApp/Models/CEvent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MyEvent.WebApp.Models
{
    public class CEvent : Data.Models.Event
    {
        public CLocationInfo oPrimaryLocation { get; set; }
        public CScheduleInfo oScheduleInfo { get; set; }

        public List<CPlannedActivity> oPlannedActivityList { get; set; }

        public CEvent(Data.Models.Event oOther)
            : base(oOther)
        {}
    }
}

File: MyEvent_Xamarin/MyEvent.ngWebApp/Client/app/models/api-event/models/CEvent.ts
/* tslint:disable */
import { CLocationInfo } from "./CLocationInfo"
import { CScheduleInfo } from "./CScheduleInfo"
import { CPlannedActivity } from "./CPlannedActivity"

export class CEvent {
		sName: string;
		idPrimaryLocationID: string;
		idScheduleInfoID: string;
		idRowID: string;

		oPrimaryLocation: CLocationInfo;
		oScheduleInfo: CScheduleInfo;
		oPlannedActivityList: CPlannedActivity[];
}
